templates/taxi/car_list.html

```diff
@@ -5,6 +5,7 @@
     Car list
     <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
       Create
+    <a href="{% url 'taxi:driver-list' %}" class="btn btn-secondary">Cancel</a>
     </a>
```

Comment:
There is an extra closing </a> tag here. It seems to be misplaced since the 'Create' link is already closed on line 7.